AmazonPublisherServicesAdapter/build.gradle.kts

```diff
@@ -35,7 +35,7 @@ android {
         minSdk = 21
         targetSdk = 33
         // If you touch the following line, don't forget to update scripts/get_rc_version.zsh
-        android.defaultConfig.versionName = System.getenv("VERSION_OVERRIDE") ?: "4.9.8.5.0"
+        android.defaultConfig.versionName = System.getenv("VERSION_OVERRIDE") ?: "4.9.8.6.0"
```

Comment:
Don't forget the `implementation("com.amazon.android:aps-sdk:9.8.6")` as well

Reply:
Good call.